contrib/natss/pkg/reconciler/dispatcher/natsschannel.go

```diff
@@ -131,10 +154,16 @@ func (r *Reconciler) Reconcile(ctx context.Context, key string) error {
 	}
 
 	// Try to subscribe.
-	if err := r.natssDispatcher.UpdateSubscriptions(c, false); err != nil {
+	failedSubscriptions, err := r.natssDispatcher.UpdateSubscriptions(c, false)
+	if err != nil {
 		logging.FromContext(ctx).Error("Error updating subscriptions", zap.Any("channel", c), zap.Error(err))
 		return err
 	}
+	natssChannel.Status.SubscribableStatus = r.createSubscribableStatus(natssChannel.Spec.Subscribable, failedSubscriptions)
+	if len(failedSubscriptions) > 0 {
+		logging.FromContext(ctx).Error("Some natss subscriptions failed to subscribe")
```

Comment:
maybe also log the len(failedSubscriptions)? 
I'm fine if you don't, but the current error message doesn't give enough info to debug...